src/services/UserService.js

```diff
@@ -0,0 +1,17 @@
+import { user } from '../database/models';
+import Queries from './Queries';
+/**
+* class for responses
```

Comment:
### Nitpick 

- Don't think that's a good description

Reply:
I added a lot more description